src/styles/variables.scss

```
@@ -0,0 +1,2 @@
+$cell-size: 100px;
+$start-day: "mon" 0 0, "tue" 1 3, "wed" 2 6, "thu" 3 9, "fri" 4 12, "sat" 5 15, "sun" 6 18;
```

For what purposes do we need to use the second number for each item? You can simplify it and calculate the necessary values 
```suggestion
$start-day: "mon" 1,  "tue" 2,  "wed" 3,  "thu" 4,  "fri" 5,  "sat" 6,  "sun" 7;
```